modules/assim.sequential/R/Remote_helpers.R

```diff
@@ -410,14 +410,14 @@ SDA_remote_launcher <-function(settingPath,
   
 }
 
-#' Remote.Sync.launcher
+#' Remote_Sync_launcher
 #'
 #' @param settingPath Path to your local setting .
 #' @param remote.path Path generated by SDA_remote_launcher which shows the path to your remote SDA run.
 #' @param PID PID generated by SDA_remote_launcher which shows the active PID running your SDA job.
 #'
 #' @export
-Remote.Sync.launcher <- function(settingPath, remote.path, PID) {
+Remote_Sync_launcher <- function(settingPath, remote.path, PID) {
   
   settings <- read.settings(settingPath)
```

Comment:
build is throwing the error 
```
checking R code for possible problems ... NOTE: Remote_Sync_launcher: no visible global function definition for 'read.settings'
```
so I think you need to add the PEcAn.settings:: namespace in front of this read.settings.